tesseract_task_composer/include/tesseract_task_composer/task_composer_graph.h

```diff
@@ -96,7 +99,7 @@ class TaskComposerGraph : public TaskComposerNode
   template <class Archive>
   void serialize(Archive& ar, const unsigned int version);  // NOLINT
 
-  void dumpHelper(std::ostream& os, const TaskComposerGraph& parent) const;
+  void dumpHelper(std::ostream& os, const TaskComposerGraph& parent, std::map<boost::uuids::uuid, TaskComposerNodeInfo::UPtr>& results_map) const;
```

Comment:
Same comment about making the map const

```suggestion
  void dumpHelper(std::ostream& os, const TaskComposerGraph& parent, const std::map<boost::uuids::uuid, TaskComposerNodeInfo::UPtr>& results_map) const;
```